browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -480,6 +481,9 @@ void RewardsDOMHandler::RegisterMessages() {
   web_ui()->RegisterMessageCallback("brave_rewards.saveOnboardingResult",
       base::BindRepeating(&RewardsDOMHandler::SaveOnboardingResult,
       base::Unretained(this)));
+  web_ui()->RegisterMessageCallback(
```

Comment:
what purpose does `brave_rewards.isInitialized` serve if we also need `pageReady`?

Reply:
this would mean that some methods are being called without checking to see if rewards is initialized. I don't think checking `IsInitialized` is a robust way to handle this is in the first place vs returning some kind of error value or delaying the response until it is, but since we are currently using `IsInitialized` it seems like it should be used consistently